CHANGES.rst

```diff
@@ -1,3 +1,8 @@
+Version 0.12.0 (2015-??-??)
+---------------------------
+
+* FEATURE: the custom filter names can now also be used for ordering #230
```

Comment:
Is this "also"? (Rather than "instead")

i.e. Is it a breaking change? If so we should say so "BREAKING CHANGE: ..."